harambe/parser/type_currency.py

```diff
@@ -3,68 +3,110 @@
 import re
 from typing_extensions import Annotated
 
-price_not_available_phrases = [
-    "price not available",
-    "unavailable price",
-    "price upon request",
-    "contact for price",
-    "request a quote",
-    "call for price",
-    "check price in store",
-    "price tbd (to be determined)",
-    "price not disclosed",
-    "out of stock",
-    "sold out",
-    "pricing not provided",
-    "not priced",
-    "currently unavailable",
-    "n/a (not available)",
-    "ask for pricing",
-    "see details for price",
-    "price coming soon",
-    "temporarily unavailable",
-    "price hidden",
-    "tdb",
-    "n/a",
-]
+price_not_available_phrases = set(
+    [
+        "price not available",
+        "unavailable price",
+        "price upon request",
+        "contact for price",
+        "request a quote",
+        "call for price",
+        "check price in store",
+        "price tbd (to be determined)",
+        "price not disclosed",
+        "out of stock",
+        "sold out",
+        "pricing not provided",
+        "not priced",
+        "currently unavailable",
+        "n/a (not available)",
+        "ask for pricing",
+        "see details for price",
+        "price coming soon",
+        "temporarily unavailable",
+        "price hidden",
+        "tdb",
+        "n/a",
+    ]
+)
+
+currency_symbols = set(
+    [
+        "$",
+        "€",
+        "£",
+        "¥",
+        "₹",
+        "₽",
+        "₩",
+        "₫",
+        "₴",
+        "₦",
+        "₪",
+        "₱",
+        "฿",
+        "₲",
+        "₭",
+        "₵",
+        "₡",
+        "₸",
+        "₺",
+        "ƒ",
+        "₳",
+        "₼",
+        "₢",
+        "₣",
+        "₥",
+        "₯",
+        "₠",
+        "₧",
+        "₰",
+    ]
+)
```

Comment:
These shouldnt be in an array

```suggestion
price_not_available_phrases = {
    "price not available",
    "unavailable price",
    "price upon request",
    "contact for price",
    "request a quote",
    "call for price",
    "check price in store",
    "price tbd",
    "price not disclosed",
    "out of stock",
    "sold out",
    "pricing not provided",
    "not priced",
    "currently unavailable",
    "n/a",
    "ask for pricing",
    "see details for price",
    "price coming soon",
    "temporarily unavailable",
    "price hidden",
    "tdb"
}

currency_symbols = {
    "$", "€", "£", "¥", "₹", "₽", "₩", "₫", "₴", "₦", "₪", "₱", "฿", "₲", "₭", 
    "₵", "₡", "₸", "₺", "ƒ", "₳", "₼", "₢", "₣", "₥", "₯", "₠", "₧", "₰"
}
```